buttons.css

```diff
@@ -0,0 +1,38 @@
+.loginbutton
+{
+	margin: 25%;
+	padding-left: 40px;
+	padding-right: 40px; 
+	padding-top: 10px;
+	padding-bottom: 10px;
+	background-color: #3090C7;
+	text-decoration: none;
+	font-size: 18px;
+	color: white;
+	border: 1px solid;
+	box-shadow: 0px 2px 10px grey;
+}
+
+.button1
```

Comment:
Odds are that as content is added to your page, you're going to start having headaches trying to manage the amount of buttons you have. There are better options to target an element with a particular index in CSS nowadays. Do you know what those might be?